tests/playwright/pages/PaymentMethodsPage.mjs

```diff
@@ -297,11 +297,18 @@ export default class PaymentMethodsPage {
     await oneyGender.click();
     await oneyDateOfBirth.click();
     await oneyDateOfBirth.type(shopper.dateOfBirth);
+    await oneyEmail.fill('');
     await oneyEmail.type(shopper.shopperEmail);
   };
 
-  confirmOneyPayment = async () => {
-    //Simulation on the Oney page
+  waitForOneyLoad = async () => {
+    //Simulation of the redirect to the Oney page
+    await this.page.waitForNavigation({
+      url: /.*staging.e-payments.oney/,
+      timeout: 15000,
+      waitUntil: 'networkidle',
```

Comment:
No need to wait for networkidle here, since we only need to check the URL redirection, and not wait until the whole page loads.